ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -105,6 +105,7 @@ def initialize(opts = {})
       @oidc_provider_metadata_url       = opts.fetch(:oidc_provider_metadata_url, nil)
       @oidc_client_id                   = opts.fetch(:oidc_client_id, nil)
       @oidc_client_secret               = opts.fetch(:oidc_client_secret, nil)
+      @oidc_redirect_uri                = opts.fetch(:oidc_redirect_uri, "#{protocol}#{servername}#{@oidc_uri}")
```

Comment:
```suggestion
      @oidc_redirect_uri                = opts.fetch(:oidc_redirect_uri, @oidc_uri)
```

Recommend to keep previous value as the default.  Forcing an absolute URL like `https://example.com/oidc` will cause issues for configs using ServerAlias and other similar configs where as using just `/oidc` will work more generally.